stories/resizable-card-layout/resizable-card-layout.stories.js

```diff
@@ -0,0 +1,505 @@
+        </div>
+    </div>
+`;
+
+resizeCardHorizontalVerticalDirection.storyName = 'Card resize in horizontal and vertical directions';
+resizeCardHorizontalVerticalDirection.parameters = {
+    docs: {
+        iframeHeight: 900,
+        storyDescription: `Cards can be resized in horizontal direction as well as vertical direction. Resize handle will expand on right border for horizontal resize and bottom border for vertical resize.
+        `
+    }
+};
+
+export const resizeCardHorizontalDirection = () =>
+    `<div id="grid1" class="fd-resizable-card-layout">
+        <div id="eight" class="fd-resizable-card-layout__item">
+            <div class="fd-resizable-card-layout__resize--vertical"></div>
+            <div class="fd-resizable-card-layout__card">
+                <div class="fd-card" role="region" aria-label="List Card Example 5">
+                    <a class="fd-card__header" tabindex="0">
+                        <div class="fd-card__header-text">
+                            <div class="fd-card__title-area">
+                                <div class="fd-card__title">Quick Links</div>
+                                <span class="fd-object-status fd-card__counter">4 of 20</span>
+                            </div>
+                        </div>
+                    </a>
+                    <div class="fd-card__content" role="group" aria-label="Card Content">
+                        <ul class="fd-list fd-list--no-border" role="list">
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--cart"></i>
+                                <span class="fd-list__title">List item 1</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--wrench"></i>
+                                <span class="fd-list__title">List item 2</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--leads"></i>
+                                <span class="fd-list__title">List item 3</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--batch-payments"></i>
+                                <span class="fd-list__title">List item 4</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="fd-resizable-card-layout__icon-wrapper">
+                    <div class="fd-resizable-card-layout__icon fd-resizable-card-layout__resize-cursor--corner"></div>
```

Comment:
There should be `fd-resizable-card-layout__resize-cursor--horizontal` class

Reply:
added &__icon-horizontal modifier class now.